DICOM/src/Microsoft.Health.DeID.SharedLib/DateShiftFunction.cs

```diff
@@ -13,21 +13,19 @@ namespace Microsoft.Health.Dicom.DeID.SharedLib
 {
     public class DateShiftFunction
     {
-        public DateShiftFunction(DateShiftSetting dateShiftSetting)
+        public DateShiftFunction(DateShiftSetting dateShiftSetting = null)
         {
-            if (dateShiftSetting != null)
-            {
-                DateShiftKey = dateShiftSetting.DateShiftKey;
-                DateShiftKeyPrefix = dateShiftSetting.DateShiftKeyPrefix;
-                DateShiftRange = dateShiftSetting.DateShiftRange;
-            }
+            dateShiftSetting ??= new DateShiftSetting();
+            DateShiftKey = dateShiftSetting.DateShiftKey;
+            DateShiftKeyPrefix = dateShiftSetting.DateShiftKeyPrefix;
```

Comment:
In cli tool document, DateShift parameters are dateShiftRange, dateShiftKey and dateShiftScope.
dateShiftScope = dateShiftKeyPrefix?
Please make document, sample configuration file and code consistent.